src/components/FormInput/FormInput.types.ts

```diff
@@ -0,0 +1,26 @@
+import type { TextFieldProps } from '@mui/material/TextField';
```

Comment:
This file references React types in the props signatures (React.ChangeEvent, React.FocusEvent) but does not import React or the specific event types. Depending on tsconfig settings, this may cause 'React' is not defined errors in a types-only file.

**Sugerencia:**
```typescript
Add an explicit type-only import. Example:

import type React from 'react';

or import specific types:

import type { ChangeEvent, FocusEvent } from 'react';

Then use ChangeEvent<HTMLInputElement> / FocusEvent<HTMLInputElement> in your prop types to avoid referencing the namespace React directly.
```